<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="GameBoard" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>900</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="CountCardDeck">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red;</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelScore1">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>200</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:yellow;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelScore2">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>280</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:yellow;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>200</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:yellow;</string>
    </property>
    <property name="text">
     <string>Очки 1 гравця</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>280</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:yellow;</string>
    </property>
    <property name="text">
     <string>Очки 2 гравця</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>240</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:yellow;</string>
    </property>
    <property name="text">
     <string>Гра до:</string>
    </property>
   </widget>
   <widget class="QLabel" name="MaxScore">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>240</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:yellow;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="CloseEnd"/>
    <addaction name="SaveAs"/>
    <addaction name="Save"/>
    <addaction name="Rules"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="SaveAs">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/image/PNG-cards/icon/saveAs.png</normaloff>:/image/PNG-cards/icon/saveAs.png</iconset>
   </property>
   <property name="text">
    <string>Зберегти як</string>
   </property>
  </action>
  <action name="Save">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/image/PNG-cards/icon/save.png</normaloff>:/image/PNG-cards/icon/save.png</iconset>
   </property>
   <property name="text">
    <string>Зберегти</string>
   </property>
  </action>
  <action name="Rules">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/image/PNG-cards/icon/rules.png</normaloff>:/image/PNG-cards/icon/rules.png</iconset>
   </property>
   <property name="text">
    <string>Правила</string>
   </property>
  </action>
  <action name="CloseEnd">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/image/PNG-cards/icon/close.png</normaloff>:/image/PNG-cards/icon/close.png</iconset>
   </property>
   <property name="text">
    <string>Закрити гру</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GameBoard</class>
   <extends>QGraphicsView</extends>
   <header>gameboard.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
